### DISCUSSION TITLE
Question: IDurableOrchestrationClient Thread-Safety

### DISCUSSION THREAD

**Discussion by mpaul31** [Azure/azure-functions-durable-extension#1760] [Q&A] [answered]
Is it safe to use a single instance of `IDurableOrchestrationClient` and start a bunch of tasks calling `StartNewAsync`? My use-case is exposing an admin endpoint for my users to request starting of one or more orchestration instances.

Thanks!

**Comment by cgillum** [comment, marked as answer]
Yes, the methods in IDurableOrchestrationClient should all be thread safe.

**Comment by mpaul31** [comment]
great thank you!
